features/event-receiver/org.wso2.carbon.event.input.adapter.file.server.feature/pom.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <parent>
+        <groupId>org.wso2.carbon.event-processing</groupId>
+        <artifactId>event-receiver-feature</artifactId>
+        <version>2.0.4-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>org.wso2.carbon.event.input.adapter.file.server.feature</artifactId>
+    <packaging>pom</packaging>
+    <name>WSO2 Carbon - Event Input HTTP Adapter Server Feature</name>
+    <version>2.0.4-SNAPSHOT</version>
```

Comment:
These versions were removed later since it was defined in the parent pom.xml. The latest pom.xml is available in carbon-analytics-common repository
https://github.com/wso2/carbon-analytics-common/blob/master/features/event-receiver/org.wso2.carbon.event.input.adapter.filetail.server.feature/pom.xml